### DISCUSSION TITLE
crazyflie-simulation : Error building C controller

### DISCUSSION THREAD

**Discussion by NageshEranki** [bitcraze/discussions#1646] [Q&A]
Hi all

I am trying out the Webots simulator for the Crazyflie. I follow [these instructions](https://www.bitcraze.io/documentation/repository/crazyflie-simulation/main/functional_areas/controllers/). I am facing the following issue:

1. The 'crazyflie_controller_c' controller in Webots results in the following error in the Webots console:

`WARNING: crazyflie_controller_c: Could not find controller file:
WARNING: crazyflie_controller_c: Expected either: crazyflie_controller_c, crazyflie_controller_c.jar, crazyflie_controller_c.class, crazyflie_controller_c.py, crazyflie_controller_c.m  or Dockerfile
WARNING: crazyflie_controller_c: Starts the <generic> controller instead.` 

After reading the [official documentation](https://cyberbotics.com/doc/guide/controller-start-up) from Webots, I realized the executable 'crazyflie_controller_c' is not in my 'controller' folder. I tried running `make` in my terminal, but that doesn't work:
`Makefile:72: /resources/Makefile.include: No such file or directory
make: *** No rule to make target `/resources/Makefile.include'.  Stop.`

Am I missing a step?

Thanks!

**Comment by Bananazooka** [comment]
Hi, which OS are you running?
